### PR TITLE
Updating Network icons

### LINKED ISSUE #2970: [UX] Update chain icons
## Description and context

Chain icons needs to be updated with the latest provided assets from UX.

### PR DESCRIPTION
**Pull request by emreboga**
This PR separates network and token icons and updates the network icons with the new designs.

Fixes https://github.com/wormhole-foundation/wormhole-connect/issues/2970

Please see for details:
https://www.loom.com/share/86efa7fbfa71468bb250ee3d0fd8ef71?sid=b8f468d4-6fed-49d0-8b38-b15c46990167

### CODE REVIEW

File: wormhole-connect/src/icons/Chains/ARBITRUM.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
...qyfJd07F0rpdCOvBuOc8/7aBh5jnmkAAAAAAADAsFFIygc4H13r7pzzdH5rR+XznlkAAAAAAADAsNQcsXGuVP4r56PrXQznv8l1VI71bAIAAAAAAAAyobg4Ni90VM7JJ+nLRmGtZb8beMwUi7GJZxAAAAAAAACQOS3Jy1vmknR8PkmXGIj1Jv1+YDhvnh2besYAAAAAAAAAmXfTgvQDuaRysSFdf9Sj/Y+J0/yJcwAAAAAAAGBEGvhq93ypPC6fpE8akEdscwbOOI+IjTwjAAAAAAAAgBFv4Ou6c6XKqYUknW9QHhGtziflm1rby/t59AMAAAAAAAC8iULSOTZfSlvySbrK0Jy5Xikk5csnL0g/4pEOAAAAAAAAsJYmL3h1+1xSviSfVJ42PA/r6v3dk+9IT25JYjOPbAAAAAAAAIB3YXJH5z6v/6n0FQbpYdOLa/60+cLuv/AIBgAAAAAAAFjHWpLK1oWO9MxCkv4qn6Q1I/WQKx34jQ6TS52HNkds7BELAAAAAAAAsB4MfMV7Pin/bT5J57z+NeEG7A3T0kJS/udcR+XY4uLY3CMTAAAAAAAAYANqSZZtmy+VxxWStOhr3tdLnfmOypSB0fyap2MLj0AAAAAAAACAIei6xa+9P5eUv9bfT/t7xti9TqrlO9K5hSS9LFcq7+/r2QEAAAAAAACGoZ8l3TsXksq3X//T6a8Yw9eqRn+L80n5+kIp/UbrI13beCQBAAAAAAAAZEzrI6/tmiulf51P0tZ/G4mdn95fTyFJ5+dL6dX5jvLXB74S3yMFAAAAAAAAYITJz+ncqtDedUguSb+T70j/pZCkv8knaV+Gx/KV+VJa6v81l08q/62QdI5tSWIzjwQAAAAAAAAA/ovi4ti8Nal8MleqnJrrqEwYGNZfH527hs9QXn6t///mefmk8vNCqfz/D/zJ8lzHa7s4vxwAAAAAAACAdWLg681zpfL+uaT8tXxS/ttCUr4831GZki+l9+aT9NH+Xhz4SvRBHMdX9fdCLkkf6f/1noGBPF+q/CCXVM7LdVSO/bc/UV7Z2k8KAAAAAAAAgCFhymNL3teSVHbIlzo/Prmjc5980vnFXFI5OpekJ/17hY7KOX/cH/9nraXKUQN/z5q/t/+fUWhPPzrwz/RvFgAAAAAAAAAAAACAQfN/AXvFzMpNXhdJAAAAAElFTkSuQmCC"
```
Comment (artursapek) [inline wormhole-connect/src/icons/Chains/ARBITRUM.tsx]: these are giant base64 blobs. why are the chain icons so much data? are we bloating the bundle unnecessarily here? we should use SVGs wherever possible, not PNGs.

Comment (emreboga) [reply wormhole-connect/src/icons/Chains/ARBITRUM.tsx]: Great point! The new icons disk size is an extra 1,636,463 bytes, that's ridiculous! Let me follow up with Chloe 👍

Comment (artursapek) [reply wormhole-connect/src/icons/Chains/ARBITRUM.tsx]: jesus christ lol

File: wormhole-connect/src/config/types.ts [hunk 2]
```diff
@@ -28,7 +28,30 @@ import RouteOperator from 'routes/operator';
 import { UiConfig } from './ui';
 import { TransferInfo } from 'utils/sdkv2';
 
-export enum Icon {
+export enum ChainIcon {
```
Comment (artursapek) [inline wormhole-connect/src/config/types.ts]: can we use the `Chain` type from the SDK instead of creating this new thing?

Comment (emreboga) [reply wormhole-connect/src/config/types.ts]: `ChainIcon` is following the same enum mechanism as `TokenIcon`, it's an existing pattern. If you like we can track this further in a separate ticket to change them both?

Comment (artursapek) [reply wormhole-connect/src/config/types.ts]: dont worry about TokenIcon. I'm obliterating that code anyway in my arbitrary token branch.

Comment (artursapek) [reply wormhole-connect/src/config/types.ts]: there is already an enum for chains, it's `Chain` in the SDK. if we can use that we should

Comment (emreboga) [reply wormhole-connect/src/config/types.ts]: Do you mean `import { Chain } from '@wormhole-foundation/sdk';`? It seems to be a string type. We can still use that with a string => icon map:
```
const iconMap: { [Chain]: React.JSX.Element } = {
  'GLMR': GLMR(),
  'AVAX': AVAX(),
  'BSC': BSC(),
  ...
};

<ChainIcon icon={chainConfig?.symbol} />
```
Or did you mean `ChainId` as the enum for chains? I wanted to double-check if what you mean is something else.

Comment (emreboga) [reply wormhole-connect/src/config/types.ts:31]: and there is `chains` from `@wormhole-foundation/sdk` as well, which seems more like it.

**Review by artursapek — APPROVED** [review]

**Pull request merged**